### PR TITLE
Create list_fibonacci.c

### PR DESCRIPTION
**Pull request by thomasina23**
list the required number of terms in the Fibonacci Series

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This program accepts an input number from the user and list out the given number of terms in the Fibonacci Series

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Added new code

### CODE REVIEW

File: C/math/list_fibonacci.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>    
+int main()    
+{    
+     int t1=0, t2=1, t3, i, terms;
```
Comment (atarax665) [inline C/math/list_fibonacci.c:4]: Use more meaningful variable names.

File: C/math/list_fibonacci.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{    
+     int t1=0, t2=1, t3, i, terms;    
+     printf("Number of terms in Fibonacci Series (min 3): ");    
+     scanf("%d",&terms);
+     
+     //print first 2 terms : 0, 1
+     printf("\n%d %d", t1, t2);
+     
+     //print 3rd and subsequent terms
+     for(i=2;i<terms;++i)
```
Comment (atarax665) [inline C/math/list_fibonacci.c:12]:
```suggestion
     for(i = 2; i < terms; ++i)
```

File: C/math/list_fibonacci.c [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+     {    
+         t3 = t1 + t2;    
+         printf(" %d", t3);    
+         t1 = t2;    
+         t2 = t3;    
+     }
+     return 0;  
+}
```
Comment (atarax665) [inline C/math/list_fibonacci.c:20]: Add sample Input/Output and time/space complexity of your code in commented form.

File: C/math/list_fibonacci.c (hunk 2 above)
Comment (atarax665) [inline C/math/list_fibonacci.c:12]:
```suggestion
     for(i = 2; i < terms; ++i)
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
Update ```C/README.md```. Refer any merged PR for better insight.

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**